packages/shared/src/telemetry/events/component-mounted.ts

```diff
@@ -11,8 +11,19 @@ type EventComponentMounted = {
   baseTheme: boolean;
 };
 
+type EventComponentMountedRaw = {
+  component: string;
+  [key: string]: boolean | string;
+};
+
 /**
  * Fired when one of the Clerk components is mounted.
+ *
+ * @param component - The name of the component.
+ * @param props - The props passed to the component. Will be filtered to a known list of props.
+ *
+ * @example
+ * telemetry.record(eventComponentMounted('SignUp', props));
  */
 export function eventComponentMounted(
```

Comment:
can we change this one to `eventPrebuiltComponentMounted` (or something similar) and expose your currently named `eventComponentMountedRaw` as `eventComponentMounted`?

Reply:
Done 👍